sysbuild/Kconfig.wifi

```diff
@@ -4,32 +4,32 @@
 
 menu "Wi-Fi"
 
-menuconfig WIFI_NRF700X
+menuconfig WIFI_NRF70
 	bool "nRF700x"
 
-if WIFI_NRF700X
+if WIFI_NRF70
```

Comment:
Whilst I'm not against this change, this does just break existing customers for no real reason. It needs migration notes adding

Reply:
It was part of the cleanup suggested in upstream review, I will add it to migration notes.